listenbrainz/webserver/static/js/src/player-pages/PlayerPage.tsx

```diff
@@ -0,0 +1,332 @@
+      (jspfTrack: JSPFTrack, index: number) => {
+        // eslint-disable-next-line no-param-reassign
+        jspfTrack.id = getRecordingMBIDFromJSPFTrack(jspfTrack);
+      }
+    );
+    this.state = {
+      playlist: props.playlist?.playlist || {},
+      recordingFeedbackMap: {},
+    };
+  }
+
+  componentDidMount(): void {
+    const { APIService, spotifyAuth } = this.context;
+    this.APIService = APIService;
+    /* Deactivating feedback until the feedback system works with MBIDs instead of MSIDs */
+    /* const recordingFeedbackMap = await this.loadFeedback();
+    this.setState({ recordingFeedbackMap }); */
+  }
+
+  getFeedback = async (mbids?: string[]): Promise<FeedbackResponse[]> => {
+    const { newAlert } = this.props;
+    const { currentUser } = this.context;
+    const { playlist } = this.state;
+    const { track: tracks } = playlist;
+    if (currentUser && tracks) {
+      const recordings = mbids ?? tracks.map(getRecordingMBIDFromJSPFTrack);
+      try {
+        const data = await this.APIService.getFeedbackForUserForRecordings(
+          currentUser.name,
+          recordings.join(", ")
+        );
+        return data.feedback;
+      } catch (error) {
+        newAlert(
+          "danger",
+          "Playback error",
+          typeof error === "object" ? error?.message : error
+        );
+      }
+    }
+    return [];
+  };
+
+  getAlbumDetails(): JSX.Element {
+    const { playlist } = this.state;
+    return (
+      <>
+        <div>Release date: </div>
+        <div>Label:</div>
+        <div>Tags:</div>
+        <div>Links:</div>
+      </>
+    );
+  }
+
+  loadFeedback = async (mbids?: string[]): Promise<RecordingFeedbackMap> => {
+    const { recordingFeedbackMap } = this.state;
+    const feedback = await this.getFeedback(mbids);
+    const newRecordingFeedbackMap: RecordingFeedbackMap = {
+      ...recordingFeedbackMap,
+    };
+    feedback.forEach((fb: FeedbackResponse) => {
+      newRecordingFeedbackMap[fb.recording_msid] = fb.score;
+    });
+    return newRecordingFeedbackMap;
+  };
+
+  updateFeedback = (
+    recordingMsid: string,
+    score: ListenFeedBack | RecommendationFeedBack
+  ) => {
+    const { recordingFeedbackMap } = this.state;
+    recordingFeedbackMap[recordingMsid] = score as ListenFeedBack;
+    this.setState({ recordingFeedbackMap });
+  };
+
+  getFeedbackForRecordingMbid = (
+    recordingMbid?: string | null
+  ): ListenFeedBack => {
+    const { recordingFeedbackMap } = this.state;
+    return recordingMbid ? get(recordingFeedbackMap, recordingMbid, 0) : 0;
+  };
+
+  handleError = (error: any) => {
+    const { newAlert } = this.props;
+    newAlert("danger", "Error", error.message);
+  };
+
+  getHeader = (): JSX.Element => {
+    const { playlist } = this.state;
+    const { track: tracks } = playlist;
+    const releaseLink =
+      tracks?.[0]?.extension?.[MUSICBRAINZ_JSPF_PLAYLIST_EXTENSION]
+        ?.release_identifier;
+    const isPlayerPage = false;
```

Comment:
Note to self: This is currently hardcoded to disable showing more album info.
When we have more metadata available to show we can revisit it.